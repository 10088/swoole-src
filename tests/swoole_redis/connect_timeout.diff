001+ Fatal error: Uncaught Error: Class 'swoole_redis' not found in /src/swoole-src/tests/include/api/swoole_redis/connect_timeout.php:2
001- 
002+ Stack trace:
003+ #0 /src/swoole-src/tests/swoole_redis/connect_timeout.php(3): require_once()
004+ #1 {main}
005+   thrown in /src/swoole-src/tests/include/api/swoole_redis/connect_timeout.php on line 2