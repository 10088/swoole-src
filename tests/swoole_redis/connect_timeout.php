<?php
require_once __DIR__ . "/../include/swoole.inc";
require_once __DIR__ . "/../include/api/swoole_redis/connect_timeout.php";

?>

