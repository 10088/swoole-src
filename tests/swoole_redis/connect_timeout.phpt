--TEST--
swoole_redis: connect timeout

--SKIPIF--
<?php require __DIR__ . '/../include/skipif.inc'; ?>
--FILE--
<?php
require_once __DIR__ . '/../include/bootstrap.php';
require_once __DIR__ . '/../include/api/swoole_redis/connect_timeout.php';

?>

--EXPECT--
