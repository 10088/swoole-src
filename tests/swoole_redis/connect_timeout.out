Fatal error: Uncaught Error: Class 'swoole_redis' not found in /src/swoole-src/tests/include/api/swoole_redis/connect_timeout.php:2
Stack trace:
#0 /src/swoole-src/tests/swoole_redis/connect_timeout.php(3): require_once()
#1 {main}
  thrown in /src/swoole-src/tests/include/api/swoole_redis/connect_timeout.php on line 2